lib/authz/permissions.go

```diff
@@ -593,6 +593,7 @@ func roleSpecForProxy(clusterName string) types.RoleSpecV6 {
 				types.NewRule(types.KindSAMLIdPServiceProvider, services.RO()),
 				types.NewRule(types.KindUserGroup, services.RO()),
 				types.NewRule(types.KindIntegration, services.RO()),
+				types.NewRule(types.KindInstance, services.RO()),
```

Comment:
Should we add a comment here to say something like "the proxy needs this to show outdated instances in the web UI" or something like that? (is that even accurate? an auditor on the proxy will just have an auditor auth client for the proxy to use)